spaces/S000209/README.md

```diff
@@ -0,0 +1,10 @@
+---
+uid: S000209
+name: Circle with two origins
+---
+
+Choose a point $0 \in S^1$ to be called the origin, and replace $0$ with two origins $0_1$ and $0_2$. Basic open neighborhoods of a point $x \neq 0$ are Euclidean open neighborhoods of $x$ not containing $0$. Basic open neighborhoods of each origin $0_i$ are of the form $(U\setminus\{0\})\cup\{0_i\}$ with $U$ a Euclidean open neighborhood of $0$.
+
+Let $\{1, 2\}$ have the discrete topology. $X$ is homeomorphic to the quotient space of $S^1 \times \{1, 2\}$ obtained by identifying $\langle \theta, 1 \rangle$ and $\langle \theta, 2 \rangle$ exactly when $\theta \not\equiv 0 \mod 2\pi$.
+
+$X$ is homeomorphic to the one-point compactification of {S83}.
```

Comment:
I don't like the name "one-point compactification" when used when referring to non-Hausdorff compactifications, since then we are lacking uniqueness. 
Instead, how about call it Alexandroff extension?